<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyGISClass</class>
 <widget class="QMainWindow" name="MyGISClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyGIS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyGIS.qrc">
    <normaloff>:/MyGIS/icons/地球.png</normaloff>:/MyGIS/icons/地球.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="MyGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_Shapefile"/>
    <addaction name="action_GeoJSON"/>
    <addaction name="action_WKT"/>
    <addaction name="separator"/>
    <addaction name="action_Raster"/>
    <addaction name="action_SBRD"/>
    <addaction name="separator"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionScsv"/>
    <addaction name="actionSdb"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionLayerManager"/>
    <addaction name="actionStatisticView"/>
    <addaction name="actionHistogramView"/>
    <addaction name="actionAttributesView"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>栅格数据分析</string>
    </property>
    <addaction name="actionColorDisplay"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionMask"/>
    <addaction name="actionNeighbor"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>矢量数据分析</string>
    </property>
    <addaction name="actionConvexHull"/>
    <addaction name="actionBuffer"/>
    <addaction name="actionDelaunay"/>
    <addaction name="separator"/>
    <addaction name="actionStatistic"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>日志</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_6"/>
   <addaction name="menu_5"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVectorLayer"/>
   <addaction name="actionRasterLayer"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>图层管理器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="MyTreeView" name="treeView">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>属性表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>矢量统计结果</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>灰度直方图</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGraphicsView" name="graphicsView_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartEditing"/>
   <addaction name="actionStopEditing"/>
   <addaction name="actionForward"/>
   <addaction name="actionBackward"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLayerManager"/>
   <addaction name="actionAttributesView"/>
   <addaction name="actionStatisticView"/>
   <addaction name="actionHistogramView"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPan"/>
   <addaction name="actionSelect"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionLeftRotate"/>
   <addaction name="actionRightRotate"/>
  </widget>
  <action name="actionVectorLayer">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/打开文档.png</normaloff>:/MyGIS/icons/打开文档.png</iconset>
   </property>
   <property name="text">
    <string>导入矢量图层</string>
   </property>
  </action>
  <action name="actionRasterLayer">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/打开文档.png</normaloff>:/MyGIS/icons/打开文档.png</iconset>
   </property>
   <property name="text">
    <string>导入栅格图层</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionStartEditing">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/执行.png</normaloff>:/MyGIS/icons/执行.png</iconset>
   </property>
   <property name="text">
    <string>开始编辑</string>
   </property>
  </action>
  <action name="actionStopEditing">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/取消.png</normaloff>:/MyGIS/icons/取消.png</iconset>
   </property>
   <property name="text">
    <string>停止编辑</string>
   </property>
  </action>
  <action name="actionLayerManager">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/图层控制.png</normaloff>:/MyGIS/icons/图层控制.png</iconset>
   </property>
   <property name="text">
    <string>图层管理器</string>
   </property>
  </action>
  <action name="actionConvexHull">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/凸包.png</normaloff>:/MyGIS/icons/凸包.png</iconset>
   </property>
   <property name="text">
    <string>凸包计算</string>
   </property>
  </action>
  <action name="actionBuffer">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/缓冲区.png</normaloff>:/MyGIS/icons/缓冲区.png</iconset>
   </property>
   <property name="text">
    <string>缓冲区计算</string>
   </property>
  </action>
  <action name="actionStatistic">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/统计.png</normaloff>:/MyGIS/icons/统计.png</iconset>
   </property>
   <property name="text">
    <string>统计矢量要素</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/直方图.png</normaloff>:/MyGIS/icons/直方图.png</iconset>
   </property>
   <property name="text">
    <string>灰度直方图</string>
   </property>
  </action>
  <action name="actionMask">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/影像专题图切图.png</normaloff>:/MyGIS/icons/影像专题图切图.png</iconset>
   </property>
   <property name="text">
    <string>按掩膜提取</string>
   </property>
  </action>
  <action name="actionNeighbor">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/栅格分析.png</normaloff>:/MyGIS/icons/栅格分析.png</iconset>
   </property>
   <property name="text">
    <string>邻域统计</string>
   </property>
  </action>
  <action name="actionColorDisplay">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/颜色通道.png</normaloff>:/MyGIS/icons/颜色通道.png</iconset>
   </property>
   <property name="text">
    <string>真假色显示</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/减速.png</normaloff>:/MyGIS/icons/减速.png</iconset>
   </property>
   <property name="text">
    <string>向前</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/加速.png</normaloff>:/MyGIS/icons/加速.png</iconset>
   </property>
   <property name="text">
    <string>向后</string>
   </property>
  </action>
  <action name="actionStatisticView">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/统计.png</normaloff>:/MyGIS/icons/统计.png</iconset>
   </property>
   <property name="text">
    <string>矢量统计表</string>
   </property>
  </action>
  <action name="actionHistogramView">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/直方图.png</normaloff>:/MyGIS/icons/直方图.png</iconset>
   </property>
   <property name="text">
    <string>灰度直方图</string>
   </property>
  </action>
  <action name="actionAttributesView">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/属性表.png</normaloff>:/MyGIS/icons/属性表.png</iconset>
   </property>
   <property name="text">
    <string>图层属性表</string>
   </property>
  </action>
  <action name="action_Shapefile">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/添加数据.png</normaloff>:/MyGIS/icons/添加数据.png</iconset>
   </property>
   <property name="text">
    <string>导入Shapefile</string>
   </property>
  </action>
  <action name="action_WKT">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/添加数据.png</normaloff>:/MyGIS/icons/添加数据.png</iconset>
   </property>
   <property name="text">
    <string>导入WKT</string>
   </property>
  </action>
  <action name="action_GeoJSON">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/添加数据.png</normaloff>:/MyGIS/icons/添加数据.png</iconset>
   </property>
   <property name="text">
    <string>导入GeoJSON</string>
   </property>
  </action>
  <action name="action_Raster">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/添加数据.png</normaloff>:/MyGIS/icons/添加数据.png</iconset>
   </property>
   <property name="text">
    <string>导入栅格数据</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/漫游.png</normaloff>:/MyGIS/icons/漫游.png</iconset>
   </property>
   <property name="text">
    <string>平移</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/选择.png</normaloff>:/MyGIS/icons/选择.png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/放大.png</normaloff>:/MyGIS/icons/放大.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/缩小.png</normaloff>:/MyGIS/icons/缩小.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_SBRD">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/添加数据.png</normaloff>:/MyGIS/icons/添加数据.png</iconset>
   </property>
   <property name="text">
    <string>导入超大栅格</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/保存.png</normaloff>:/MyGIS/icons/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存工程</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/打开.png</normaloff>:/MyGIS/icons/打开.png</iconset>
   </property>
   <property name="text">
    <string>打开工程</string>
   </property>
  </action>
  <action name="actionLeftRotate">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/icon_顺时针旋转.png</normaloff>:/MyGIS/icons/icon_顺时针旋转.png</iconset>
   </property>
   <property name="text">
    <string>顺时针旋转</string>
   </property>
  </action>
  <action name="actionRightRotate">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/icon_逆时针旋转.png</normaloff>:/MyGIS/icons/icon_逆时针旋转.png</iconset>
   </property>
   <property name="text">
    <string>逆时针旋转</string>
   </property>
  </action>
  <action name="actionScsv">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/文本.png</normaloff>:/MyGIS/icons/文本.png</iconset>
   </property>
   <property name="text">
    <string>存储为wkt</string>
   </property>
  </action>
  <action name="actionSdb">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/数据库,数据.png</normaloff>:/MyGIS/icons/数据库,数据.png</iconset>
   </property>
   <property name="text">
    <string>存储为数据库</string>
   </property>
  </action>
  <action name="actionDelaunay">
   <property name="icon">
    <iconset resource="MyGIS.qrc">
     <normaloff>:/MyGIS/icons/构建网络.png</normaloff>:/MyGIS/icons/构建网络.png</iconset>
   </property>
   <property name="text">
    <string>Delaunay三角网</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>MyGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>MyTreeView</class>
   <extends>QTreeView</extends>
   <header>MyTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyGIS.qrc"/>
 </resources>
 <connections/>
</ui>
